datalad/tests/test_utils.py

```diff
@@ -1305,8 +1316,17 @@ def get_interactive(py_pre="", **run_kwargs):
         assert_in(out, bools)
         return bool(bools.index(out))
 
-    # we never request for pty in our Runner, so can't be interactive
-    eq_(get_interactive(), False)
+    # verify that NoCapture can make fully interactive execution
+    # happen, also test the core protocols
+    # (we can only be interactive in a runner, if the test execution
+    # itself happens in an interactive environment)
+    for proto, interactive in ((NoCapture, is_interactive()),
+                               (KillOutput, False),
+                               (StdOutErrCapture, False),
+                               (GitProgress, False),
+                               (AnnexInitOutput, False),
+                               (AnnexJsonProtocol, False)):
+        eq_(get_interactive(protocol=proto), interactive)
```

Comment:
I am not sure what is happening on travis with this test. https://travis-ci.org/github/datalad/datalad/jobs/734538506#L2302 says that while the test execution itself considers `is_interactive() == False`, a process executed by the test is `is_interactive() == True`. This seems wrong. I hand-verified that the logic works on my Debian system, and on Win10.

Reply:
Added a special case handling for travis.